float64[] desired
---
bool status